runtime/onert/core/include/ir/Subgraphs.h

```diff
@@ -131,6 +132,8 @@ class Subgraphs
 
 private:
   std::unordered_map<SubgraphIndex, std::shared_ptr<Graph>> _subgraphs;
+  ModelOperandIndexSequence _inputs;
```

Comment:
This might be redundant since we assume that 1st Graph's inputs(and outputs) == Model's inputs(outputs) ?

Reply:
@chunseoklee It was copied from #6274 (which we've agreed to be a base). I also would like to remove if you and @hseok-oh are okay.